packages/app/utils/time.ts

```diff
@@ -1,8 +1,13 @@
-export const currentTimestampInSeconds = Math.floor(Date.now() / 1000);
```

Comment:
Suggestion for this file name: `datetime.ts` or `date-time.js`

Reply:
👍

Reply:
datetime.js (I think in this case makes sense to be together)